### PR TITLE
DOC: minor grammar fixes in minimize and KDTree.query

### LINKED ISSUE #13950: DOC: scipy.spatial.KDTree.query
In this paragraph, a tabular layout in the source is run together without punctuation in the generated docs:

> Which Minkowski p-norm to use. 1 is the sum-of-absolute-values “Manhattan” distance 2 is the usual Euclidean distance infinity is the maximum-coordinate-difference distance A large, finite p may cause a ValueError if overflow can occur.

### PR DESCRIPTION
**Pull request by Ennosigaeon**
Fixes a small typo in the minimize docs.

Fixes #13950.

### CODE REVIEW

File: scipy/optimize/_minimize.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
 
         where ``x`` is an array with shape (n,) and ``args`` is a tuple with
         the fixed parameters. If `jac` is a Boolean and is True, `fun` is
-        assumed to return and objective and gradient as an ``(f, g)`` tuple.
+        assumed to return an objective and gradient as a ``(f, g)`` tuple.
```
Comment (peterbell10) [inline scipy/optimize/_minimize.py:99]: I read this as "eff-gee tuple" which starts with a vowel sound, so "an" was correct.

**Comment by tupui** [comment]
Hi, thanks for contributing to SciPy.

We usually discourage very small fixes like these as explained in our contributing documentation: https://scipy.github.io/devdocs/dev/hacking.html#maintaining-existing-code
The reasoning behind it being that every contribution needs to be reviewed and incur maintainers (we are all volunteers) time and effort.

Could you maybe find more mistakes to have something more substantial to contribute here?

**Comment by peterbell10** [comment]
I'm not sure that note applies here. This isn't code-style (which has no impact on end users), it's grammar in the public documentation.

**Comment by tupui** [comment]
> I'm not sure that note applies here. This isn't code-style (which has no impact on end users), it's grammar in the public documentation.

Well we might re-phrase a bit this. IMO this is still very small though and we could not have thousands of such things as this would quickly become unsustainable.

File: scipy/optimize/_minimize.py (hunk 1 above)
Comment (WarrenWeckesser) [reply scipy/optimize/_minimize.py:99]: I agree with @peterbell10--"an" *sounds* better to me--but the phrase `as an ``(f, g)`` tuple` is a bit awkward.  As long as we're editing this line to fix the grammatical mistake, we might consider a more substantial copy-edit, perhaps something like
```
        assumed to return a tuple of length two holding the objective function
        and the gradient.
```

**Comment by WarrenWeckesser** [comment]
@tupui, we definitely don't want a lot of subjective, "tomayto or tomahto"-type copy-editing.  But fixing an unambiguous grammatical error, even if the change is tiny, is always worthwhile.

**Comment by tupui** [comment]
@Ennosigaeon this is something you could add to your PR #13950

File: scipy/optimize/_minimize.py (hunk 1 above)
Comment (Ennosigaeon) [reply scipy/optimize/_minimize.py]: I have changed the sentence as suggested by @WarrenWeckesser.

**Comment by Ennosigaeon** [comment]
Just let me know how to proceed with this PR. If you want to I can close it and submit it again if I have collected more changes.

**Review by rgommers — APPROVED** [review]
Thanks @Ennosigaeon. It looks good now, so let's just merge it.

**Comment by rgommers** [comment]
>  If you want to I can close it and submit it again if I have collected more changes.

This closed the issue, so is good as is. If you plan to do a lot of copy-edit like changes, it would indeed be nice to collect them in a larger PR.

**Pull request merged**